templates/download/alternative-downloads.html

```diff
@@ -41,23 +54,56 @@ <h2 id="bittorrent">BitTorrent</h2>
     <div class="col-4">
       <h3 class="p-heading--4"><span>Ubuntu {{ releases.latest.full_version }}</span></h3>
       <ul class="p-list--divided">
-        <li class="p-list__item"><a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.latest.short_version }}/ubuntu-{{ releases.latest.full_version }}-desktop-amd64.iso.torrent">Ubuntu {{ releases.latest.full_version }} Desktop (64-bit)</a></li>
-        <li class="p-list__item"><a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.latest.short_version }}/ubuntu-{{ releases.latest.full_version }}-live-server-amd64.iso.torrent">Ubuntu Server {{ releases.latest.full_version }}</a></li>
+        <li class="p-list__item">
+          {# <a class="download-torrent"
+            href="https://releases.ubuntu.com/{{ releases.latest.short_version }}/ubuntu-{{ releases.latest.full_version }}-desktop-amd64.iso.torrent">Ubuntu
+            {{ releases.latest.full_version }} Desktop (64-bit)</a> #}
+          <p class="p-form-help-text u-no-margin--bottom" id="downloadButtonHelpMessage">Ubuntu 23.10 Desktop cannot be
```

Comment:
This helper text was added and the link above was commented out. That's the only change in this file, the rest is also an auto-formatting by IDE